cmd/node/factory/structs.go

```diff
@@ -1275,9 +1276,12 @@ func newShardBlockProcessorAndTracker(
 	if err != nil {
 		return nil, nil, err
 	}
-	//TODO: change the mock
+
+	cryptoHook := &hooks.VMCryptoHook{}
```

Comment:
should make a constructor for those hooks.

Reply:
Right, changed everywhere in the project.